internal/provider/workflow_job_resource_test.go

```diff
@@ -281,9 +281,16 @@ func TestAccAAPWorkflowJob_Basic(t *testing.T) {
 }
 
 func TestAccAAPWorkflowJobWithNoInventoryID(t *testing.T) {
-	jobTemplateID := os.Getenv("AAP_TEST_WORKFLOW_INVENTORY_ID")
+	jobTemplateID := os.Getenv("AAP_TEST_WORKFLOW_JOB_TEMPLATE_ID")
 	inventoryID := os.Getenv("AAP_TEST_INVENTORY_FOR_WF_ID")
 
+	if jobTemplateID == "" {
+		t.Skip("AAP_TEST_WORKFLOW_JOB_TEMPLATE_ID not set, skipping workflow job test")
+	}
+	if inventoryID == "" {
+		t.Skip("AAP_TEST_INVENTORY_FOR_WF_ID not set, skipping workflow job inventory test")
+	}
+
```

Comment:
Please remove these changes. We should not be skipping tests. Also, what's the reason for changing the environment variable? Was this not working before?